### PR TITLE
fix scripts

### PR DESCRIPTION
**Pull request by joshuarli**
a few small tweaks, in particular one fix for `update.sh` not working; json response from pypi was one line, so need to emit line-delimited json for awk to work as expected.

### CODE REVIEW

**Comment by mattrobenolt** [comment]
👀

**Comment by JoshuaRLi** [comment]
lol hi matt

File: build-git.sh [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
 set -e
 
 usage () {
     echo "usage: $0 <sha>"
     exit 1
 }
 
-if [ "$#" = 0 ]; then
-    set -- "$(curl -sSL 'https://api.github.com/repos/getsentry/sentry/git/refs/heads/master' | awk -F '"' '$2 == "sha" { print $4 }')"
+if (( $# == 0 )); then
```
Comment (mattrobenolt) [inline build-git.sh]: Do you run a weird version of bash or something? What was the issue?

Comment (JoshuaRLi) [reply build-git.sh]: no issue here; i generally prefer using arithmetic evaluation for `$#` with bash scripts

**Comment by BYK** [comment]
Superseded by #177 as this PR introduces unrelated changes. #177 also keeps using `awk` instead of using Python to format the JSON output (less dependencies? 😀 )

**Pull request closed without merging**